src/utils/useResponsive.js

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from 'react';
+
+export const useResponsive = () => {
+    const [res, setRes] = useState(window.innerWidth);
+
+    useEffect(() => {
+        window.addEventListener('resize', () => setRes(window.innerWidth));
```

Comment:
DRY, don't repeat yourself, if you are using same function more than one, you should define it as separate function, 

Also please handle the fontFize of root in mediaQueries in scope of this task

Reply:
done